tests/deploy.sh

```diff
@@ -1,7 +1,51 @@
 #!/bin/bash
 
 # Skip deploy for staging branch
-[ "$TRAVIS_BRANCH" = "staging" ] && exit 0
+[ "$BRANCH" = "staging" ] && exit 0
 
 docker login -u $DOCKER_UN -p $DOCKER_PW
+
+if [ "$BRANCH" = "testing" ]
+then
+  docker-compose -f tests/build.yml push
+  exit 0
+fi
+
+#Deploy for main releases
+#Images are built with tag PINNED_MAILU_VERSION (x.y.z).
+#We are tagging them as well with MAILU_VERSION (x.y)
+#After that, both tags are pushed to the docker repository.
+if [ "$PINNED_MAILU_VERSION" != "" ] && [ "$BRANCH" != "master" ]
+then
+  images=$(docker-compose -f tests/build.yml config | grep 'image: ' | awk -F ':' '{ print $2 }')
```

Comment:
you don't need an extra grep. awk can do this like this:
```
awk -F ':' '/image:/{ print $2 }'
```

Reply:
I will give it a try. It is better to have less piping if possible.